dataverse-webapp/src/main/webapp/dataset.xhtml

```diff
@@ -37,7 +37,6 @@
         </ui:define>
         <ui:define name="jsonld_header">
             <script type="application/ld+json">
-                <h:outputText value="#{DatasetPage.jsonLd}"/>
             </script>
         </ui:define>
```

Comment:
Question: why we're removing this?

Reply:
Ahh, since this line is not treated as proper in IJ it was cleaned up, since I have "Cleanup" option when committing, I have to disable it.